### ISSUE TITLE
Gets super busy at midnight for some reason

### ISSUE THREAD

**Issue by WatchfulRelic91** [dymanoid/RealTime#262]
I have no clue why everybody is going shopping at midnight, very unrealistic and annoying when citizens should be sleeping. Any idea why? I was thinking maybe day and night are mixed up somehow but i don't know how i would fix that. Any tips would be appreciated. 
![yes ok](https://user-images.githubusercontent.com/62899919/77939396-1b275200-7285-11ea-81be-d7bc205ff966.png)

**Comment by dymanoid** [comment]
Interestingly enough, there are other players reporting that it is too boring for them in the night - #221.
This heavily depends on your city layout and your Real Time configuration.
Have you tried inspecting the routes and the citizens - where are all they going and why?

**Comment by WatchfulRelic91** [comment]
Yes. A lot of them are either going home or going shopping, both are strange and unrealistic because they should have gone home around 6 which is what it is set for. This midnight rush is also much  busier than the other 2 rushes, vary annoying. I could screenshot my realtime settings if you would like, but everything schedule based is default. I could also send mod list.

**Comment by dymanoid** [comment]
And where the citizens are going *from*?

Not everyone goes back at 6, because there are three shifts. If you e.g. have a lot of industry, then a lot of workers will work there second and night shift.

Shopping at night is fully okay too, especially for people going back from work.

**Comment by WatchfulRelic91** [comment]
I do have a considerable amount of industry. should i reduce the second and/or night shift probability? would that reduce the intensity of it? Or would that even work?

**Comment by dymanoid** [comment]
I think you should identify the problem first. Inspect where are all the citizens are going *from* (you can additionally use my **Snooper** mod which will show you the exact building where the citizens are going from). Also, observe how long they are under way. You can also check which work shift the citizens have. And so on. It is hard to guess - it is your city, you should investigate it, because you built it.

**Comment by WatchfulRelic91** [comment]
I've been snooping around during the midnight rush, and it seems that a lot of these people are not industrial workers, and many are unemployed. They just go from shop to shop, then home after the rush is over, doesn't seem to have much to do with shopping after work. Still no clue what's going on. I also found another thread about the same thing here https://github.com/dymanoid/RealTime/issues/174 at the end of it you say that you are fixing the issue. It seems that his problems are the same as mine, so i'm curious as to what the issue was and what the fix was. If it was a bug, maybe it's back after the sunset harbor update because i don't remember having this issue at all before the update.

**Comment by Sipke82** [comment]
Related to #221, there i did some research on the busyness and you can see that between the rush hours it's equally busy then during the nights...
I also recognize this behavior, its very annoying, in fact it made me unsubscribe from RealTime (sadly).

Perhaps it would be an idea to make relevant parameters customization in a XML? so the more advance users can customize for their needs?

**Comment by WatchfulRelic91** [comment]
I may have to unsubscribe as well. Also, it seems that dymanoid has been ignoring the issue for some time now. I even tried opening an identical thread to get his attention as seen above, but he just closed it and did nothing further. Very sad.

**Comment by dymanoid** [comment]
I just checked it by myself. I loaded my showcase city to test whether there is something wrong with the citizens' schedules.
I didn't find anything suspicious.

My parks are empty at night.
![Cities Skylines Screenshot 2020 04 07 - 19 57 48 75](https://user-images.githubusercontent.com/9433345/78705271-891aeb80-790d-11ea-8275-b4f89f8c8184.png)

Only a couple of citizens go shopping (no mass shopping at night):
![Cities Skylines Screenshot 2020 04 07 - 20 05 37 59](https://user-images.githubusercontent.com/9433345/78705384-ac459b00-790d-11ea-8ccb-e1c6972048fc.png)

Most citizens either go to work because of night shift or go shopping because they work second shift:
![Cities Skylines Screenshot 2020 04 07 - 20 07 22 98](https://user-images.githubusercontent.com/9433345/78705529-e616a180-790d-11ea-91ef-a413b20352bd.png)

And I spotted only one senior at 1 a.m. - he was on his way back home and was late because of a long journey.

Also, I spotted no children at that time.

So if your cities are fully crowded at 3 a.m., I can only suspect a mod conflict or an unfortunate city configuration. There is nothing wrong with Real Time as it is.

I've seen on the Workshop page that people are starting complaining about too many citizens at night. It seems to me that some mod was updated (or just a new mod appeared) and it now conflicts with Real Time - that leads to this behavior.

Again, I tested with Real Time and no other mods - it's working as intended. Can you disable all your other mods and see if things change?

**Comment by WatchfulRelic91** [comment]
Thank you for taking the time to respond (sorry for assuming you would not). Will disable mods, however it will take a very long time to test all of the mods i use, here's a list, I use a compatibility checker to make sure that they are all compatible.

81 tiles (fixed for c:s 1.2+)
AVO 1.9.0
Auto Repair
Customize it 1.6.2
Demand master
Electric roads mod 2
Extra landscaping tools
Find it 1.5.4
First person camera
Heavy outside traffic (Set to low traffic)
Hide it
Infinite goods
Islntercity fix
Klyte's fine road anarchy 2.0.2
Klyte's fine road tool 2.0.4
Loading screen mod
More network stuff
Move it 2.7.1
Network Extensions 2
Plop growables
Ploppable asphalt
Precision Engineering
Prefab Hook
Random tree rotation
Real time
Roads united core 2.0
Snooper
Stops and stations
Surface painter
TM:PE 11.2.2 Stable
Total Autobudget
Unlimited outside connections
Unlock all + wonders and landmarks
Watch it
WG Realistic population v8.4.0

Being a coder, maybe you have some ideas of what could be conflicting with your mod.

**Comment by DaEgi01** [comment]
If some of you use Realistic Walking Speed (RWS), keep in mind that RWS increases walking time by a factor of ~2.5. That could cause the issue since CIMs need a lot longer to get to where ever they want to go. Couple that with a big city where CIMs go to the other part of town for shopping and things can get messy at night.

**Comment by Sipke82** [comment]
Good point Egi, perhaps i should decrease the max walking distance in the Global TMPE parameters xml file.

**Comment by Sipke82** [comment]
Well i did test it again, but my city is still a bit small, 10k..

It was now indeed not as busy at night anymore... however still very boring during  the day, (between rushhours), thus unsubscribing again :-(

Perhaps its an idea to add some kind of busynis setting?

**Comment by BenjaminHoegh** [comment]
i do also have issues with the time in this mod, for some reason it ignore changes i made in the options, also have to agree that there are more cars doing the night
